swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md

```diff
@@ -179,6 +186,9 @@ The `mysql:Client` provides two primary remote methods for performing queries.
 Use `query()`, `queryRow()` and `execute()` methods to define methods that can perform basic CRUD operations against the MySQL database.
 
 ```ballerina
+import ballerina/sql;
```

Comment:
```suggestion
import ballerina/sql;
```
Don't we need to remove this unused import?

Reply:
The unused import was in a previous sample. It has been removed from there.
https://github.com/ballerina-platform/ballerina-dev-website/pull/5041/files#diff-13e9db3064102e2d1ec06a2f2b92cbdb93f195684ceb41c21418864b92813937L152